### ISSUE TITLE
why use "mean" in the merge_aug_bboxes function?

### ISSUE THREAD

**Issue by gfjiangly** [open-mmlab/mmdetection#1252]
https://github.com/open-mmlab/mmdetection/blob/381f491776a3e934be88c895a3f17f256342c629/mmdet/core/post_processing/merge_augs.py#L40-L64

in bboxes = torch.stack(recovered_bboxes).mean(dim=0), If the detected bbox position of the multi scales are not matched, is it not correct if the bbox is not detected correctly?

BTW, how to merge bboxes should be decided by the user?  For example, I want to combine the detection results of different cropped images.

**Comment by gfjiangly** [comment]
Can the merge function be designed as a class, and then specified in the configuration file, the user can design his own merge function to inherit from the mmdet's merge class

**Comment by hellock** [comment]
Thanks for your advice! We will redesign the TTA apis in the future refactoring.
